src/extension.ts

```diff
@@ -12,6 +12,7 @@ import {
 } from "vscode-extension-telemetry-wrapper";
 import { AddStartersHandler, GenerateProjectHandler } from "./handler";
 import { getTargetPomXml, loadPackageInfo } from "./Utils";
+import { ProjectType } from "./model";
```

Comment:
Linting failed. 

`ERROR: (ordered-imports) src/extension.ts[15, 1]: Import sources within a group must be alphabetized.`

Reply:
Sorry for that @testforstephen! Fixed it.